Sources/OpenId4VP/AuthenticationResponse/AuthenticationResponse.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+public struct AuthenticationResponse {
+    let response: [String: String]
+    
+    static func getAuthenticationResponse(_ authorizationRequest: AuthorizationRequest,_ trustedVerifierJSON: [[Verifier]], openId4VpInstance: OpenId4VP) throws -> AuthenticationResponse {
+        
+        Logger.getLogTag(className: String(describing: self))
```

Comment:
how traceabilityId will be attached to this logger?

Reply:
I am setting the trace id in the authenticate verifier method which will set trace id of the logger class,
And in all other structs we are updating the struct name to the logger class.
this way trace id is set once and used by logger methods in all other structs.